test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -48,8 +48,10 @@ class TestPreemptParamsQmgr(TestInterfaces):
     error_unauth = "Unauthorized Request"
     error_code_unauth = "15007"
```

Comment:
Same here .

Reply:
We are not checking unauthorized request error for User now .
Is that code removed intentionally or by mistake ?

Reply:
by mistake :).